.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - id: docker-org-and-tag
         uses: usdot-fhwa-stol/actions/docker-org-and-tag@main
         with:
-          docker_tag_suffix: humble
+          docker_tag_suffix: ""
```

Comment:
Oh sorry I just realized it has not been working before. Then we don't need this whole with: argument I think.